AlfiyaSiddique/TastyTrails
# Fix users comments.

## What does this PR do?



Fixes #332

updated the comments schema to refer to the userId commenting it.
also updated the controller of the add comments and deletecomments to take in the userID for proccessing the functions..
update the authenticateToken middleware to return the userId after verifying the tastytoke.


## How should this be tested?

- go to the recipe section
- try to delete other person comments.
- you u will see error.

## Checklist



### Required
